### ISSUE TITLE
Custom instance for services?

### ISSUE THREAD

**Issue by hellojaccc** [SimonBrazell/privacy-redirect#290]
It'd be nice to use a custom private instance with this extension.

**Comment by hellojaccc** [comment]
![sfsfdsfsdfssdfsfsd](https://user-images.githubusercontent.com/71537512/132003888-dcb04900-b986-42c1-b6e6-155cf0236227.png)
Until this feature gets implemented, you can use private instance with this

**Comment by xanoni** [comment]
So you're requesting that the user can enter the instance in the "General" menu instead of overwriting the pool in the "Advanced" menu?

I agree that this would make sense from an UX perspective, but it's probably not the number 1 priority, given most people just use the default instances and there's this workaround for others.

**Comment by Tyil** [comment]
This would be very nice to have, especially since I don't see a field to overwrite the pool for Reddit alternatives under the advanced menu.

**Comment by NateTheSage** [comment]
I think if we use the Bibliogram random instance selector as a template, as it is the simplest, we could easily implement the rest, there'd just be some features missing out.

I think I'll start a PR with these features, if I can get it to work for myself.

**Comment by xanoni** [comment]
@NateTheSage Yeah ... but note that there's a big backlog of other outstanding PRs that might conflict with what you're planning... so maybe can build on top of one of those. I myself have 4 PRs pending that haven't been looked at really.

**Comment by ManeraKai** [comment]
Added that logic here: https://github.com/libredirect/libredirect/commit/85aadc0eb6988e3c2e3e8d504ebf84bbf9180759
btw this is a forked repo: https://github.com/libredirect/libredirect
